src/plugin/regions/index.js

```diff
@@ -134,10 +134,9 @@ export default class RegionsPlugin {
         this._onBackendCreated = () => {
             this.wrapper = this.wavesurfer.drawer.wrapper;
             this.orientation = this.wavesurfer.drawer.orientation;
+            this.defaultEdgeScrollWidth = this.wrapper.clientWidth * scrollWidthProportion;
```

Comment:
Will this still work if the container is resized? (E.g. when resizing the window)
It could be fine if it doesn't, maybe we can just add a comment.

Reply:
I've added a comment.

An alternative approach would have been for this calculation to be done in the relevant bit of onMove in region.js if no value for edgeScrollWidth had been set. That would guarantee it would be up-to-date, although I'd worry that getting the value of clientWidth out of an HTML element might be one of those things that's surprisingly expensive on some browsers, and the effect of doing that for every mouse movement (whilst a region was being dragged) would be noticeable.